shenyu-spi/src/test/java/org/apache/shenyu/spi/ExtensionLoaderTest.java

```diff
@@ -183,10 +183,6 @@ public void testGetExtensionLoaderCanNotInstantiatedSPI() {
             ExtensionLoader.getExtensionLoader(JdbcSPI.class).getJoin("canNotInstantiated");
             fail();
         } catch (IllegalStateException expected) {
-            assertThat(expected.getMessage(), containsString(
```

Comment:
remove the test, it is meaningless to remain the code without assert

Reply:
ok, I will delete

Reply:
I suggest we can keep this line.
`"Extension instance(name: canNotInstantiated, class: class org.apache.shenyu.spi.fixture.CanNotInstantiatedSPI)  "`

Reply:
agree

Reply:
ok, this message, the jdk8 and jdk11 are the same